web/packages/teleterm/src/ui/TopBar/Identity/IdentityList/IdentityList.tsx

```diff
@@ -81,6 +81,27 @@ export function IdentityList(props: {
         </Box>
       </KeyboardArrowsNavigation>
     </Box>
+export function ClusterList(props: {
+  clusters: Cluster[];
+  onSelect(clusterUri: RootClusterUri): void;
+  onLogout?(clusterUri: RootClusterUri): void;
+  onAdd(): void;
+}) {
+  return (
+    <>
+      {props.clusters.map((cluster, index) => (
+        <IdentityListItem
+          key={cluster.uri}
+          index={index}
+          cluster={cluster}
+          onSelect={() => props.onSelect(cluster.uri)}
+          onLogout={
+            props.onLogout ? () => props.onLogout(cluster.uri) : undefined
+          }
+        />
+      ))}
+      <AddClusterItem index={props.clusters.length + 1} onClick={props.onAdd} />
```

Comment:
I'm not 100% sure, but shouldn't index be `props.clusters.length`? If `props.clusters` has 4 items, there will be four `IdentityListItem` components with indices from 0 to 3. It seems that `AddClusterItem` should then have an index of 4, but this code will make it have index of 5.

Reply:
Right 🤦